index.html

```diff
@@ -15,14 +15,254 @@
   <link href="https://fonts.googleapis.com/css?family=Gloria+Hallelujah|Varela+Round" rel="stylesheet">
   <!-- main css -->
   <link rel="stylesheet" type="text/css" href="css/style.css" />
-  <!-- Font -->
-  <link href="https://fonts.googleapis.com/css?family=Muli:400,700" rel="stylesheet">
   <!-- Animate CSS -->
   <link rel="stylesheet" href="css/animate.css">
-
 </head>
   <body>
+    <!-- START NAV AND BANNER FEAT#12, BUILD BY ZUBAIR -->
+    <!-- start feat: nav -->
+    <nav class="navbar navbar-expand-lg sticky-top nav-header">
+      <div class="container">
+        <!-- strat logo -->
+        <a class="navbar-brand" href="#">
+          <img src="img/logo.png" alt="Lunatic">
+        </a>
+        <!-- end logo -->
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
```

Comment:
Bug: Toggler icon is not displayed.
Fix: Use hamburger icon from other repository instead of boostrap standard.

Reply:
Solved!